config/server.py

```diff
@@ -101,6 +101,9 @@
 config_lib.DEFINE_integer("ACL.approvers_required", 2,
                           "The number of approvers required for access.")
 
+config_lib.DEFINE_integer("ACL.token_expiry", 10080,
```

Comment:
This should probably be defined in seconds and defaulted in a similar way to the statsstore TTL with 60 * 60 * 24 * 7